### DISCUSSION TITLE
Integrating with an existing app (Bublr)

### DISCUSSION THREAD

**Discussion by solomonshalom** [steven-tey/novel#401] [Q&A]
Hi!

Hope your all doing great!

I am curious to know if novel can be added to an existing app?

I and my team are currently building an open-source social networking site, [Bublr](https://bublr.life) and would love to integrate novel as part of it but not really sure on the how 😅 

Would love to know!

Thanks!

P.S. The repository for Bublr can be found [here](https://github.com/solomonshalom/bublr) :D

**Comment by MODSetter** [comment]
Yes it can be easily integrated. LMK if you need any help regarding it.

**Reply by solomonshalom** [reply]
I see! 

Yes! I would love some guidance on the integration, not really sure on how to go about it 😅 

P.S. Bublr does have Tip-Tap integrated, so it should be relatively easy to add Novel to it, right? 🤔

**Reply by MODSetter** [reply]
Raise a feature request in Bulbr repo. Will raise a PR soon 👍

**Reply by solomonshalom** [reply]
> Raise a feature request in Bulbr repo. Will raise a PR soon 👍

Absolutely, Thank You! 😁 

You can find the feature request here: [https://github.com/solomonshalom/Bublr/issues/68](https://github.com/solomonshalom/Bublr/issues/68) :D

**Reply by MODSetter** [reply]
> > Raise a feature request in Bulbr repo. Will raise a PR soon 👍
> 
> Absolutely, Thank You! 😁
> 
> You can find the feature request here: [solomonshalom/Bublr#68](https://github.com/solomonshalom/Bublr/issues/68) :D

Will close it by end of weekend

**Reply by solomonshalom** [reply]
OH! Thank You @MODSetter 😁!

**Reply by solomonshalom** [reply]
Hi! @MODSetter,

Hope you're doing great!

Have you had a chance to look at it 😅 

In any case, thank you so much for offering to take it up - means a lot!

**Reply by MODSetter** [reply]
> Hi! @MODSetter,
> 
> Hope you're doing great!
> 
> Have you had a chance to look at it 😅 
> 
> In any case, thank you so much for offering to take it up - means a lot!

Hi @solomonshalom 

In all honesty last week I was busy with MVP of https://github.com/MODSetter/next-blog

As I had to submit it to some hackathon.

Give me a day or two I will work on it 👍

**Reply by MODSetter** [reply]
Hi @solomonshalom 
Can you add me on LinkedIn: https://www.linkedin.com/in/rohan-verma-sde/
Lets talk.

**Reply by solomonshalom** [reply]
> > Hi! @MODSetter,
> > 
> > Hope you're doing great!
> > 
> > Have you had a chance to look at it 😅 
> > 
> > In any case, thank you so much for offering to take it up - means a lot!
> 
> Hi @solomonshalom 
> 
> In all honesty last week I was busy with MVP of https://github.com/MODSetter/next-blog
> 
> As I had to submit it to some hackathon.
> 
> Give me a day or two I will work on it 👍

OH! I completely understand!

Thank You!

P.S. I did check out the repository yesterday and it looks pretty cool! ATB w/ the hackathon 😁

**Reply by solomonshalom** [reply]
> Hi @solomonshalom 
> Can you add me on LinkedIn: https://www.linkedin.com/in/rohan-verma-sde/
> Lets talk.
> 

Sure!
